<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>576</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>576</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Sisyphus : Progress Window</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../icon/sisyphus.png</normaloff>../icon/sisyphus.png</iconset>
  </property>
  <widget class="QWidget" name="mainLayout">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPlainTextEdit" name="progressBox">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::TextSelectableByMouse</set>
      </property>
      <property name="placeholderText">
       <string>The progress output will be displayed here</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="ActionLayout">
      <item>
       <widget class="QToolButton" name="hideButton">
        <property name="minimumSize">
         <size>
          <width>275</width>
          <height>64</height>
         </size>
        </property>
        <property name="text">
         <string>Hide Window</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../icon/progress.png</normaloff>../icon/progress.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
        <property name="arrowType">
         <enum>Qt::NoArrow</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="clearButton">
        <property name="minimumSize">
         <size>
          <width>275</width>
          <height>64</height>
         </size>
        </property>
        <property name="text">
         <string>Clear Window</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../icon/orphans.png</normaloff>../icon/orphans.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>progressBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
